### PR TITLE
LPS-131934 Modify Asset Publisher configuration info message to specify the max number of items to be display

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/display_settings.jsp [hunk 1]
```diff
@@ -51,7 +51,7 @@ PortletURL configurationRenderURL = (PortletURL)request.getAttribute("configurat
 	<aui:option label="view-in-context" selected="<%= assetPublisherDisplayContext.isAssetLinkBehaviorViewInPortlet() %>" value="viewInPortlet" />
 </aui:select>
 
-<aui:input helpMessage='<%= LanguageUtil.get(request, "number-of-items-to-display-help") + ' ' + LanguageUtil.format(request, "maximum-allowed-value-is-x", SearchContainer.MAX_DELTA, false) %>' label="number-of-items-to-display" name="preferences--delta--" type="text" value="<%= assetPublisherDisplayContext.getDelta() %>">
+<aui:input helpMessage='<%= LanguageUtil.get(request, "number-of-items-to-display-help") + " " + LanguageUtil.format(request, "maximum-allowed-value-is-x", SearchContainer.MAX_DELTA, false) %>' label="number-of-items-to-display" name="preferences--delta--" type="text" value="<%= assetPublisherDisplayContext.getDelta() %>">
```
Comment (ealonso) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/display_settings.jsp:54]: I think it should be StringPool.SPACE

Comment (ealonso) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/display_settings.jsp:54]: cc: @jkappler

Comment (robertoDiaz) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/display_settings.jsp:54]: In other cases I see this pattern @ealonso

**Comment by jkappler** [comment]
Just started reviewing :)

**Comment by jkappler** [comment]
Just started reviewing :)

**Comment by jkappler** [comment]
https://github.com/liferay-echo/liferay-portal/pull/4582

**Pull request closed without merging**